.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
please remove this file from git and add to .gitignore

Reply:
done